docs/06-documentation-and-release-notes/files/README-for-Demos.md

```diff
@@ -0,0 +1,130 @@
+# README.md Template for Demos
+
+# [Connector_Name] Anypoint&trade; Studio Connector Demo - [Demo_Name]
```

Comment:
Sure about this? Can't we just remo the "Studio" part? Demo names don't include any reference to the connector in their names.

* Anypoint Connector for Demo - Leads and Programs CRUD

* Marketo Anypoint Connector Demo -  Leads and Programs CRUD